docs/modules/ROOT/pages/advanced-how-tos/using-trusted-artifacts.adoc

```diff
@@ -0,0 +1,38 @@
+= Using Trusted Artifacts
+
+Konflux relies on Trusted Artifacts to securely allow sharing of data between Tasks without
+compromising the build process. See
+link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[ADR-36] for more
+information.
+
+Today, when a component is onboarded to Konflux, the provided build Pipeline uses Trusted Artifacts.
```

Comment:
We don't technically have the ability to create components with the UI with this project since the project does not have a way to automatically create the required image repositories.  In order to be consistent with this documentation, we would have to provide the way for users to specify the pipeline in their gitops repo. Can we just state the name of the pipeline to select/configure?

If a pipeline isn't specified by default in the gitops flow, will the build service automatically select the OCI-TA pipeline by default or will it fall back to the older pipeline?

Reply:
Hmm. I realize that I don't understand how onboarding works when the UI is not involved. Is there a "send PR" workflow outside the UI?